### DISCUSSION TITLE
the defaultValues don't let the form to be updated

### DISCUSSION THREAD

**Discussion by Cuantico** [react-hook-form/react-hook-form#5136] [Q&A]
Why if you prepopulate the form using the defaultValues (in order to pass validations) in useForm and defaultValue on the textField (in order to show the data in input field) and then the information in the inputs field is updated and submitted, these change is lost and the original defaulValues are sent again?

**Comment by bluebill1049** [comment]
because the idea for `defaultValues` is made is to populate input's defaultValue, it's not defaultState, but this PR should improve it by register those missing inputs: https://github.com/react-hook-form/react-hook-form/pull/5069
